tests/SampleApps/php/greetings/composer.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "frob/greetings",
+    "description": "This is a simple hello world example.",
+    "authors": [
+        {
+            "name": "Frank Anderson",
+            "email": "frob@249517.no-reply.drupal.org"
+        }
+    ],
+    "minimum-stability": "dev",
+    "require": {
+        "php": ">=5.3.0",
+        "phpunit/php-timer": "^1.0"
+    },
+    "autoload": {
+        "psr-0": {
+            "HelloWorld": "src//"
```

Comment:
changed this from original app …  in original app it was 
`
"autoload": {

        "psr-0": {

            "HelloWorld": "src/"

        },

        "classname": {

          "PHP_Timer": "src/"

        }

    },
`